platform/mds/mds/src/main/java/org/motechproject/mds/repository/internal/AllEntityDrafts.java

```diff
@@ -68,7 +68,7 @@ public void setProperties(EntityDraft draft, Entity entity, String username) {
 
         draft.setDraftOwnerUsername(username);
         draft.setLastModificationDate(DateUtil.nowUTC());
-
+        draft.setSuperClass(entity.getSuperClass());
```

Comment:
Do we really need to set this field?  I checked it and it worked without this line.